its/src/test/java/com/sonar/it/csharp/CoverageTest.java

```diff
@@ -33,13 +33,13 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @ExtendWith(Tests.class)
-public class CoverageTest {
+class CoverageTest {
 
   @TempDir
   private static Path temp;
 
   @Test
-  public void should_not_import_coverage_without_report() throws Exception {
+  void should_not_import_coverage_without_report() throws Exception {
```

Comment:
Educational: why cannot we follow what you did in a previous file and remove the `throws Exception` checked exceptions for this method and the subsequent ones?
```suggestion
  void should_not_import_coverage_without_report() {
```

Reply:
It depends on the method implementation. In this case, the `analyzeCoverageTestProject` method throws `IOException`.